frappe/templates/includes/breadcrumbs.html

```diff
@@ -6,12 +6,12 @@
 			{% for parent in parents %}
 				<li itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem" class="breadcrumb-item">
 					<a itemprop="item" href="{{ url_prefix }}{{ parent.route | abs_url }}" itemprop="url">
-						<span itemprop="name">{{ parent.title or parent.label or parent.name or "" }}</span>
+						<span itemprop="parent">{{ parent.title or parent.label or parent.name or "" }}</span>
```

Comment:
Where is this coming from? Is there any example of this on the web?

The current implementation is from https://schema.org/BreadcrumbList